unfoldingWord/translationCore
# Add VerseCheck

#### This pull request addresses:

This pull request adds VerseCheck to the repo. It also removes CommentBox and ProposedChanges.


#### How to test this pull request:

Initalize the VerseCheck submodule, and be sure that tN and tW are on the latest versions of develop. Also, a PR is pending on tW for this to work with that tool. Open each tool, make sure that that does not fail, and then make sure the versecheck tool is there.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/931)
